usr.sbin/bsdinstall/scripts/docsinstall

```diff
@@ -148,7 +148,7 @@ f_mustberoot_init
 dialog_menu_main || f_die
 f_dialog_menutag_fetch selected
 
-# Let pkg_add be able to use name servers
+# Let pkg-add be able to use name servers
```

Comment:
In the `bsdinstall` context, what's the proper way to rethink this comment? 

Whilst I _can_ imagine something using name servers, I can _not_ imagine it being anything, from a package server, that precedes a bootstrap by [pkg(7)](https://man.freebsd.org/cgi/man.cgi?query=pkg&sektion=7&manpath=freebsd-release) in base.

(Does that make sense?)

Thanks @igalic for encouraging me to think twice about this PR.